dockerfiles/init/manifests/che.pp

```diff
@@ -28,8 +27,26 @@
   $microsoft_client_id = getValue("CHE_MICROSOFT_CLIENT_ID","your_microsoft_client_id")
   $microsoft_secret = getValue("CHE_MICROSOFT_SECRET","your_microsoft_secret")
 
-###############################
-# Include base module
+  ###############################
+  # Http proxy configuration
+  # leave those fields empty if no configuration needed
+  #
+  # http proxy for CHE
+  $http_proxy_for_che = getValue("CHE_HTTP_PROXY_FOR_CHE","")
```

Comment:
But what is the problem to use CHE_HTTP_PROXY variable? Please elaborate because I don't understand